### ISSUE TITLE
http issue with stripe integration and credit card update info

### ISSUE THREAD

**Issue by wrought** [sudoroom/sudo-humans#56]
`https://sudoroom.org/humans/~user/edit/sudoroom`

user demonstrated issue to me. Everything under the header starting at the `<div id="content>` is blank, no good console or network debugging information. Server-side log:

```
http.js:689
    throw new Error('Can\'t set headers after they are sent.');
          ^
Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (http.js:689:11)
    at /home/sudohumans/sudo-humans/lib/layout.js:16:17
    at error (/home/sudohumans/sudo-humans/bin/cmd.js:268:11)
    at /home/sudohumans/sudo-humans/routes/payment.js:71:39
    at null._onTimeout (/home/sudohumans/sudo-humans/node_modules/stripe/lib/StripeResource.js:89:34)
    at Timer.listOnTimeout [as ontimeout] (timers.js:110:15)
```

**Comment by wrought** [comment]
@Hainish @Juul @substack

**Comment by jnny** [comment]
Replicated this error... Running Ubuntu 14.04, attempted to edit credit card info on Firefox, Chrome, & Chromium and received blank page.

**Comment by Juul** [comment]
Fixed by 5cb54cf16124de515c8e4a686dc2b0d73f4f9102
